pages/api/swagger-to-actions.ts

```diff
@@ -55,9 +55,23 @@ export default async function handler(
     // Allow info.version = null(!!)
     if ("info" in swagger)
       swagger.info.version = swagger.info.version ?? "1.0.0";
+    if (swagger.swagger && swagger.swagger.startsWith("2.")) {
+      res.status(400).json({
+        message: "Swagger version 2 not supported",
+        // This format is used to display errors in the UI
+        error: {
+          "1": {
+            message:
+              "Swagger version 2 not supported: convert to version 3: https://stackoverflow.com/questions/59749513/how-to-convert-openapi-2-0-to-openapi-3-0",
+          },
+        },
+      });
+      return;
+    }
     // @ts-ignore
     dereferencedSwagger = await SwaggerParser.validate(swagger, {
       dereference: { circular: "ignore" },
+      validate: { spec: false },
```

Comment:
This handles openapi 2 & sends helpful error message to frontend when uploading
![Screenshot from 2023-07-28 10-50-17](https://github.com/Superflows-AI/superflows/assets/33871096/e0004b69-53cf-49d1-8609-920985997697)